Dev/jgpoland/io 4532 api docs housekeeping

GimmeYerGold opened the pull request:
CHANGES

• removed 8080 dev comment
• fixed errors addressed by api markdown linter
• revised some response and request examples

Commit:
clearing up errors and removing old notes

Commit:
Merge branch 'master' into dev/jgpoland/io-4532-api-docs-housekeeping

Commit:
[api revisions] updated syntax

rmoscowitz commented on api/endpoints/move.md:
```diff
@@ -626,11 +629,13 @@ To update a Move Request send JSON with updated values for one or more of the at
 
 
-## Move Collection [/move/requests///{{?statusId,statusCode}}]
+## Move Collection [/move/requests/{?statusId,statusCode}]
```
Does this guy still work with only one `/`?

rmoscowitz reviewed (commented):
Looks good! Only one comment.

Commit:
[chores] i found a lot of little fixes lol

GimmeYerGold commented:
@rmoscowitz i addressed your comment by escaping *all* curly braces like so `///{thingy}`

I also found a lot of little other things that needed tweaking 😄

Commit:
[code review] fixed missing Move Collection params

rmoscowitz reviewed (approved):
LGTM!
